src/shared.js

```diff
@@ -292,23 +292,21 @@ export const performSearch = async (query) => {
     if (response.code === 200 && verifiedParticipants.length > 0) {
       searchResults(verifiedParticipants);
     } else if (response.code === 200 && verifiedParticipants.length === 0) {
-      showNotifications(
-        {
-          title: 'Not found',
-          body: 'The participant with entered search criteria not found!'
-        },
-        true
-      );
+      showNotifications({title: 'Not found', body: 'The participant with entered search criteria not found!'});
     }
 }
 
-export const showNotifications = (data, error) => {
+// show notifications in the UI. z-index controls the order of the modal
+// zIndex of 1000 is less than loading animation z-index of 9999. set zIndex to 10000 to show notifications on top of loading animation.
+// Current use: shimpent completion success notification. Show the success modal above the loading animation.
+export const showNotifications = (data, zIndex) => {
```

Comment:
[Reference to this showNotifications call ](https://github.com/episphere/biospecimen/pull/587/files#diff-aa3209e98840e6d236562086e3c6942a28f0c6beef6897d036367d09cc313c59R2137-R2138)
Nice implementation of `z-index` in the showNotifications function.